airbyte-commons/src/main/java/io/airbyte/commons/json/JsonSchemas.java

```diff
@@ -189,8 +189,7 @@ private static void traverseJsonSchemaInternal(final JsonNode jsonSchemaNode,
             // hit every node.
             traverseJsonSchemaInternal(jsonSchemaNode.get(JSON_SCHEMA_ITEMS_KEY), newPath, consumer);
           } else {
-            throw new IllegalArgumentException(
-                "malformed JsonSchema array type, must have items field in " + jsonSchemaNode);
+            log.warn("The array is missing an items field. The traversal is silently stopped. Current schema: " + jsonSchemaNode);
```

Comment:
This does not only affect arrays without items, it also affects the OBJECT_TYPE exception thrown below (`throw new IllegalArgumentException(
                "malformed JsonSchema object type, must have one of the following fields: properties, oneOf, allOf, anyOf in " + jsonSchemaNode);`)

Could we do the same there?

Reply:
Do we have object without properties?

Reply:
Yes. This was the case for github here https://github.com/airbytehq/airbyte/pull/15049

For APIs it can be common for a stream to have an object without defined properties, for example consider an api that lets you set any arbitrary tags as an object for a record

Reply:
Done